docs_src/security/Ch-Configuring-Add-On-Services.md

```diff
@@ -139,7 +138,6 @@ the `known secret` and `device-virtual` is the service key of the add-on service
     container_name: edgex-secretstore-setup
     depends_on:
     - security-bootstrapper
-    - vault
```

Comment:
It still depends on `secret-store`

https://github.com/edgexfoundry/edgex-compose/blob/1a73aa25e39506ae4cfc0b16fccf8c526cd55f26/docker-compose.yml#L1116